src/components/ListView.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const ListView = () => {
+const ListView = ({ token }) => {
```

Comment:
Again: the token is never used in this file, isn't it?

Reply:
The token prop has been added in each view only because we knew the other team would need to use it. Will delete it now. Thank you.